src/pages/ImportWorkflow.js

```diff
@@ -47,7 +47,7 @@ const DockstoreImporter = ({ path, version }) => {
     withErrorReporting('Error loading WDL'),
     Utils.withBusyState(setIsBusy)
   )(async () => {
-    const wdl = await Ajax(signal).Dockstore.getWdl(path, version)
+    const wdl = await Ajax(signal).Dockstore.getWdl(path, version, source === 'dockstoretools')
```

Comment:
TOL the constant string`'dockstoretools'` is re-typed multiple times in multiple places. Is it worth making a global value to store it?

Reply:
It's a fair consideration, though in my experience so far, doing that mostly just increases verbosity without adding much extra safety, so I'm inclined to leave it.